querybook/webapp/components/QueryViewReview/QueryViewReview.tsx

```diff
@@ -0,0 +1,368 @@
+import { ConfirmationMessage } from 'components/ConfirmationManager/ConfirmationMessage';
+import { Icon } from 'ui/Icon/Icon';
+
+import './QueryViewReview.scss';
+import { QueryReviewState } from 'hooks/useQueryReview';
+
+const rejectSchema = Yup.object().shape({
+    rejectionReason: Yup.string()
+        .required('Please provide a reason for requesting changes')
+        .min(10, 'Reason must be at least 10 characters'),
+});
+
+const ReviewHeader: React.FC<{
+    status: Status;
+    tooltip: string;
+    label: string;
+    tagClass: string;
+    tagText: string;
+}> = memo(({ status, tooltip, label, tagClass, tagText }) => (
+    <div className="review-header">
+        <div className="header-left">
+            <StatusIcon status={status} tooltip={tooltip} />
+            <AccentText weight="bold" className="header-label">
+                {label}
+            </AccentText>
+        </div>
+        <Tag
+            className={tagClass}
+            tooltip={tooltip}
+            tooltipPos="up"
+            color={
+                status === Status.success
+                    ? 'green'
+                    : status === Status.error
+                    ? 'red'
+                    : 'yellow'
+            }
+            withBorder
+        >
+            {tagText}
+        </Tag>
+    </div>
+));
+
+const ReviewContent: React.FC<{
+    review: IQueryReview;
+}> = memo(({ review }) => (
+    <div className="review-content">
+        <div className="details-grid">
+            <div className="detail-item">
+                <AccentText color="light" size="small">
+                    Requested By
+                </AccentText>
+                <UserBadge uid={review.requested_by} />
+            </div>
+            <div className="detail-item">
+                <AccentText color="light" size="small">
+                    Reason Provided by Author
+                </AccentText>
+                <AccentText className="request-reason">
+                    {review.request_reason || 'No reason provided'}
+                </AccentText>
+            </div>
+            <div className="detail-item">
+                <AccentText color="light" size="small">
+                    Review Requested At
+                </AccentText>
+                <AccentText className="request-reason">
+                    {generateFormattedDate(review.created_at)}
+                </AccentText>
+            </div>
+        </div>
+    </div>
+));
+
+const ReviewActions: React.FC<{
+    showRejectForm: boolean;
+    onShowRejectForm: (show: boolean) => void;
+    onApprove: () => void;
+    onReject: (values: { rejectionReason: string }) => void;
+}> = memo(({ showRejectForm, onShowRejectForm, onApprove, onReject }) => {
+    const [showApproveModal, setShowApproveModal] = useState(false);
+
+    return (
+        <div className="review-actions">
+            <div className="action-buttons">
+                <Button
+                    title="Request Changes"
```

Comment:
why use "Request Changes" instead of "Reject"? 
"Request Changes" seems to be the same review request will be sent back again, while the current design is a new request will be created iirc?